src/library/Uno.Material/Styles/Controls/v2/RadioButton.xaml

```diff
@@ -1,203 +1,257 @@
-								 Fill="{TemplateBinding Background}"
-								 IsHitTestVisible="False"
-								 Opacity="0" />
-
-						<Ellipse x:Name="HoverAndPressRing"
-								 HorizontalAlignment="Center"
-								 VerticalAlignment="Center"
-								 Width="{StaticResource RadioStatesAreaSize}"
-								 Height="{StaticResource RadioStatesAreaSize}"
-								 Fill="{TemplateBinding Background}"
-								 IsHitTestVisible="False"
-								 Opacity="0" />
-
-						<Grid Height="{StaticResource RadioCheckAreaSize}"
-							  Width="{StaticResource RadioCheckAreaSize}">
-
-							<Ellipse x:Name="OuterEllipse_Checked"
-									 HorizontalAlignment="Stretch"
-									 VerticalAlignment="Stretch"
-									 Stretch="Fill"
-									 UseLayoutRounding="False"
-									 Stroke="{TemplateBinding Background}"
-									 StrokeThickness="{StaticResource RadioBorderThickness}"
-									 Opacity="0" />
-
-							<Ellipse x:Name="OuterEllipse_Unchecked"
-									 HorizontalAlignment="Stretch"
-									 VerticalAlignment="Stretch"
-									 Stretch="Fill"
-									 UseLayoutRounding="False"
-									 Stroke="{TemplateBinding BorderBrush}"
-									 StrokeThickness="{StaticResource RadioBorderThickness}" />
-
-							<Ellipse x:Name="CheckEllipse"
-									 HorizontalAlignment="Stretch"
-									 VerticalAlignment="Stretch"
-									 Stretch="Fill"
-									 UseLayoutRounding="False"
-									 Fill="{TemplateBinding Background}"
-									 Opacity="0"
-									 Margin="5" />
-						</Grid>
-
-						<ContentPresenter x:Name="ContentPresenter"
-										  ContentTemplate="{TemplateBinding ContentTemplate}"
-										  ContentTransitions="{TemplateBinding ContentTransitions}"
-										  Content="{TemplateBinding Content}"
-										  Margin="{TemplateBinding Padding}"
-										  HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
-										  VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
-										  AutomationProperties.AccessibilityView="Raw"
-										  TextWrapping="Wrap"
-										  Grid.Column="1" />
-					</Grid>
-				</ControlTemplate>
-			</Setter.Value>
-		</Setter>
-	</Style>
-
-	<Style x:Key="MaterialSecondaryRadioButtonStyle"
-		   TargetType="RadioButton"
-		   BasedOn="{StaticResource MaterialRadioButtonStyle}">
-
-		<Setter Property="Background"
-				Value="{ThemeResource SecondaryBrush}" />
-	</Style>
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+                    xmlns:android="http://uno.ui/android"
+                    xmlns:ios="http://uno.ui/ios"
+                    xmlns:wasm="http://uno.ui/wasm"
+                    mc:Ignorable="ios android wasm">
+
+    <ResourceDictionary.ThemeDictionaries>
+		<ResourceDictionary x:Key="Default">
+			<!--#region Foreground-->
+            <StaticResource x:Key="RadioButtonForeground" ResourceKey="OnBackgroundMediumBrush" />
```

Comment:
Is the Foreground actually OnSurfaceMediumBrush by default in the Figma file as well? Interesting, I wouldn't thought it would just be OnSurfaceBrush at the normal default state

Reply:
This is what I can see in Figma (all states, even disabled):
![image](https://github.com/unoplatform/Uno.Themes/assets/70542961/e7a523fa-845d-453e-a00f-97d77425b3ee)

For disabled I set it to OnBackgroundDisabledBrush, should it be OnBackgroundMediumBrush too (in Figma it's MediumBrush like the others, but I thought it looked a bit weird).